schema/rdl_schema_0.1.json

```diff
@@ -1796,7 +1678,164 @@
+        "exposure_id",
+        "vulnerability_id"
+      ],
+      "properties": {
+        "hazard_type": {
+          "title": "Hazard type",
+          "type": "string",
+          "description": "The main type of hazard involved in the modelled scenario(s), from the closed [hazard type codelist]https://rdl-standard.readthedocs.io/en/{{version}}/references/codelists.html#hazard_type).",
+          "codelist": "hazard_type.csv",
+          "openCodelist": false,
+          "enum": [
+            "coastal_flood",
+            "convective_storm",
+            "drought",
+            "earthquake",
+            "extreme_temperature",
+            "flood",
+            "landslide",
+            "tsunami",
+            "volcanic",
+            "wildfire",
+            "strong_wind"
+          ]
+        },
+        "hazard_process": {
+          "title": "Hazard process",
+          "type": "string",
+          "description": "The main hazard process involved in the modelled scenario(s), from the closed [hazard process type codelist]https://rdl-standard.readthedocs.io/en/{{version}}/references/codelists.html#hazard_process_type).",
+          "codelist": "hazard_process_type.csv",
+          "openCodelist": false,
+          "enum": [
+            "coastal_flood",
+            "storm_surge",
+            "tornado",
+            "agricultural_drought",
+            "hydrological_drought",
+            "meteorological_drought",
+            "socioeconomic_drought",
+            "primary_rupture",
+            "secondary_rupture",
+            "ground_motion",
+            "liquefaction",
+            "extreme_cold",
+            "extreme_heat",
+            "fluvial_flood",
+            "pluvial_flood",
+            "groundwater_flood",
+            "snow_avalanche",
+            "landslide_general",
+            "landslide_rockslide",
+            "landslide_mudflow",
+            "landslide_rockfall",
+            "tsunami",
+            "ashfall",
+            "volcano_balllistics",
+            "lahar",
+            "lava",
+            "pyroclastic_flow",
+            "wildfire",
+            "extratropical_cyclone",
+            "tropical_cyclone"
+          ]
+        },
+        "category": {
+          "title": "Exposure category",
+          "description": "The category of the exposed assets, from the closed [exposure category codelist](https://rdl-standard.readthedocs.io/en/{{version}}/references/codelists/#exposure_category).",
+          "type": "string",
+          "codelist": "exposure_category.csv",
+          "openCodelist": false,
+          "enum": [
+            "Agriculture",
+            "Buildings",
+            "Infrastructures",
+            "Population",
+            "Natural environment"
+          ]
+        },
+        "cost": {
+          "title": "Asset cost",
+          "type": "array",
+          "description": "The exposure costs associated with specific elements of assets detailed in the dataset.",
```

Comment:
Is 'exposure costs' the correct term here? If I understood correctly, loss data is derived from a combination of hazard, exposure and vulnerability data, i.e. the cost (value) of assets that are exposed to a hazard could be greater than the cost (value) of assets that are actually lost, depending on the vulnerability function that is applied. As such, I think the description should be something more like:

```suggestion
          "description": "The costs associated with the loss of specific elements of assets detailed in the dataset.",
```